### PR TITLE
app-text/nfoview-1.27 version bump

### PR DESCRIPTION
**Pull request by tillschaefer**
- migrate to EAPI 7 
- add gettext dependency
- remove versions not tested against python 3.7

### CODE REVIEW

File: app-text/nfoview/nfoview-1.27.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1 gnome2-utils xdg-utils
```
Comment (juippis) [inline app-text/nfoview/nfoview-1.27.ebuild]: gnome2-utils isnt' needed here anymore, and using it for icon_cache_update on an EAPI-7 ebuild shouldnt even work. Use `xdg_icon_cache_update` instead. R

File: app-text/nfoview/nfoview-1.27.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+fi
+
+DESCRIPTION="Simple viewer for NFO files, which are ASCII art in the CP437 codepage"
+HOMEPAGE="https://otsaloma.io/nfoview/"
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE=""
+
+BDEPEND="${PYTHON_DEP}
```
Comment (juippis) [inline app-text/nfoview/nfoview-1.27.ebuild]: There's a typo here, but you are right in putting it into BDEPEND.

File: app-text/nfoview/nfoview-1.27.ebuild (hunk 1 above)
Comment (tillschaefer) [reply app-text/nfoview/nfoview-1.27.ebuild]: ok, good to know. Is changed now.

File: app-text/nfoview/nfoview-1.27.ebuild (hunk 2 above)
Comment (tillschaefer) [reply app-text/nfoview/nfoview-1.27.ebuild]: done

**Pull request closed without merging**